crates/rolldown_binding/src/options/binding_output_options/binding_chunk_file_names.rs

```diff
@@ -0,0 +1,65 @@
+use anyhow::anyhow;
```

Comment:
Actually, you could normalize other types into a function, and only use the `ThreadSafeFunction` as option type in rust side, 
https://github.com/rolldown/rolldown/blob/c9f8af403fa17316f1a1c0f9c5ad437b417f74e1/packages/rolldown/src/options/bindingify-input-options.ts#L29-L48, this could reduce tons of work for binding.

Reply:
I'm ok for it, because it will avoid call js side function for string option...